readme.md

```diff
@@ -108,6 +108,7 @@ Please visit [this link](https://give.do/fundraisers/stand-beside-the-victims-of
     - [Donut Vertical Chart Language Card Layout](#donut-vertical-chart-language-card-layout)
```

Comment:
You also need to add new query string parameter information in table on 404 line

Reply:
Sorry, I did not check enough.
Add it at https://github.com/anuraghazra/github-readme-stats/pull/3708/commits/27da644f14eb1a0cd8daa6557c14e54bc4637a64.